src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Simplification/CSharpSimplification.cs

```diff
@@ -14,6 +14,6 @@ internal sealed class CSharpSimplification : AbstractSimplification
     public override SimplifierOptions DefaultOptions
         => CSharpSimplifierOptions.Default;
 
-    public override SimplifierOptions GetSimplifierOptions(IOptionsReader options, SimplifierOptions? fallbackOptions)
-        => new CSharpSimplifierOptions(options, (CSharpSimplifierOptions?)fallbackOptions);
+    public override SimplifierOptions GetSimplifierOptions(IOptionsReader options)
+        => new CSharpSimplifierOptions(options, fallbackOptions: null);
```

Comment:
don't quite understand this.  is this a hole?  (not blocking), just wondering if it represents a possible bug taht could potentially be observable.

Reply:
The fallbacks are not needed anymore since we'll get the right values from the FallbackAnalyzerOptions of the solution. 
It doesn't matter what value do we pass in here anymore. For now, this is set to null to limit the size of the PR. Will remove the parameter entirely in following PR.